.meshrc.yaml

```diff
@@ -163,7 +163,7 @@ sources:
                     "accession_name": "Hubei mosquito virus 2 strain mosZJ35453 segment 1 hypothetical protein 1 and hypothetical protein 2 genes, complete cds",
```

Comment:
this may be more of a general question about mutations, do we have to provide a request sample, or can we provide a json schema for the request?  If we can, it seems like it would be more consistent to do that,